src/Robo/Commands/Deploy/DeployCommand.php

```diff
@@ -535,11 +521,17 @@ protected function setMultisiteFilePermissions($perms) {
    */
   protected function commit() {
     $this->say("Committing artifact to <comment>{$this->branchName}</comment>...");
+    $author_string = '';
+    $git_name = $this->getConfigValue('git.user.name');
+    $git_email = $this->getConfigValue('git.user.email');
+    if ($git_name && $git_email) {
+      $author_string = sprintf('--author "%s <%s>"', $git_name, $git_email);
+    }
     $result = $this->taskExecStack()
       ->dir($this->deployDir)
       ->exec("git rm -r --cached --ignore-unmatch .")
       ->exec("git add -A")
-      ->exec(["git commit --quiet -m", escapeshellarg($this->commitMessage)])
+      ->exec(["git commit --quiet -m $author_string", escapeshellarg($this->commitMessage)])
```

Comment:
Why are you passing the author as message ? Did you want to use --author ?

Reply:
Whoops, I got the order of the arguments backwards.